python/lsst/pipe/tasks/selectImages.py

```diff
@@ -283,6 +283,31 @@ class PsfWcsSelectImagesConfig(pipeBase.PipelineTaskConfig,
         default=0.22,
         optional=True,
     )
+    minNPsfStarPerBand = pexConfig.DictField(
+        keytype=str,
+        itemtype=float,
+        default={
+            "u": 10,
+            "g": 22,
+            "r": 22,
+            "i": 22,
```

Comment:
Why 10/22? Is it to allow a threshold with the outlier rejection and based on what you observed with the weird PSF fit?

Reply:
Yup...left some buffer for outlier rejection.  This _should_ be a no-op when DM-49688 is merged.